### ISSUE TITLE
[Bug]: Too much space after webR block in RevealJS

### ISSUE THREAD

**Issue by dlsun** [coatless/quarto-webr#102]
### Bug description

There is so much whitespace after a webR block in RevealJS that any remaining content overflows off the initial viewport.

For example, the output of the minimal Quarto document below is:

<img width="1200" alt="Screenshot 2023-11-17 at 12 13 20 AM" src="https://github.com/coatless/quarto-webr/assets/1175502/09203fea-2dac-41e9-b520-ade2e4df62fc">

### Steps to reproduce

You should be able to reproduce it with the following minimal Quarto document.

````
---
title: Testing
engine: knitr
format: 
  revealjs:
    resources:
    - webr-serviceworker.js
    - webr-worker.js
webr:
  channel-type: "automatic"
filters:
  - webr
---

## Slide with webR {.scrollable}

```{webr-r}
2 + 3
```

This text should appear right below the webR block, but is way
down in 0.4.0.
````

Compare the results of running:

- `quarto add coatless/quarto-webr@0.3.8`
- `quarto add coatless/quarto-webr` (production)

### Your environment

- MacOS Sonoma 14.0
- Using RStudio

**Comment by dlsun** [comment]
I think this bug is probably caused by #98 but am not sure

**Comment by coatless** [comment]
Fixed in #103. 

Thanks! Sorry about that. Artifact of some font-size experiments
